hedera-node/hedera-consensus-service-impl/src/test/java/com/hedera/node/app/service/consensus/impl/test/codecs/ConsensusServiceStateTranslatorTest.java

```diff
@@ -0,0 +1,205 @@
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.consensus.impl.test.codecs;
+
+import static com.hedera.node.app.service.consensus.impl.ConsensusServiceImpl.TOPICS_KEY;
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertFalse;
+
+import com.hedera.hapi.node.base.Key;
+import com.hedera.hapi.node.base.TopicID;
+import com.hedera.hapi.node.state.consensus.Topic;
+import com.hedera.node.app.service.consensus.impl.codecs.ConsensusServiceStateTranslator;
+import com.hedera.node.app.service.consensus.impl.test.handlers.ConsensusTestBase;
+import com.hedera.node.app.service.mono.state.merkle.MerkleTopic;
+import com.hedera.node.app.spi.fixtures.state.MapWritableKVState;
+import com.hedera.node.app.spi.state.WritableKVState;
+import com.swirlds.merkle.map.MerkleMap;
+import edu.umd.cs.findbugs.annotations.NonNull;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.extension.ExtendWith;
+import org.mockito.junit.jupiter.MockitoExtension;
+
+@ExtendWith(MockitoExtension.class)
+public class ConsenusServiceStateTranslatorTest extends ConsensusTestBase {
+
+    @BeforeEach
+    void setUp() {}
+
+    @Test
+    void createMerkleTopicFromTopic() {
+        final var existingTopic = readableStore.getTopic(topicId);
+        assertFalse(existingTopic.deleted());
+
+        final com.hedera.node.app.service.mono.state.merkle.MerkleTopic convertedTopic =
+                ConsensusServiceStateTranslator.pbjToState(topic);
+
+        assertMatch(convertedTopic, getExpectedMonoTopic());
+    }
+
+
+
+    @Test
+    void createMerkleTopicFromTopicWithEmptyKeys() {
+        final var existingTopic = readableStore.getTopic(topicId);
+        assertFalse(existingTopic.deleted());
+
+        final com.hedera.node.app.service.mono.state.merkle.MerkleTopic convertedTopic =
+                ConsensusServiceStateTranslator.pbjToState(topicNoKeys);
+
+        assertMatch(convertedTopic, getExpectedMonoTopicNoKeys());
+    }
+
+    @Test
+    void createMerkleTopicFromReadableTopicStore() {
+        final com.hedera.node.app.service.mono.state.merkle.MerkleTopic convertedTopic =
+                ConsensusServiceStateTranslator.pbjToState(topicId, readableStore);
+
+        assertMatch(convertedTopic, getExpectedMonoTopic());
+    }
+
+    @Test
+    void createTopicFromMerkleTopic() {
+        com.hedera.node.app.service.mono.state.merkle.MerkleTopic merkleTopic =
+                getMerkleTopic( (com.hedera.node.app.service.mono.legacy.core.jproto.JKey) com.hedera.node.app.service.mono.pbj.PbjConverter.fromPbjKey(topic.adminKeyOrElse(Key.DEFAULT))
```

Comment:
You can probably import the PbjConverter and shorten this line (which might fix the Spotless error)

Reply:
I prefer all from mono to bring the full path